specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlServer.json

```diff
@@ -1541,7 +1545,6 @@
           "type": "boolean",
           "x-ms-mutability": [
```

Comment:
Is it intentional that this field couldn't read?

refs: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-mutability